interface/main/mobile/m_functions.php

```diff
@@ -0,0 +1,283 @@
+                <a <?php
+                if ($display =='flow') {
+                    echo ' class="active" ';
+                } ?>
+                        href="<?php echo $GLOBALS['webroot']; ?>/interface/patient_tracker/patient_tracker.php?skip_timeout_reset=1"><?php echo xlt('Flow'); ?></a>
+                <?php if ($GLOBALS['medex_enable'] =='1') { ?>
+                <a <?php
+                if ($display =='sms') {
+                    echo ' class="active" ';
+                } ?>
+                        href="<?php echo $GLOBALS['webroot']; ?>/interface/main/messages/messages.php?nomenu=1&go=SMS_bot&dir=back&show=new"><?php echo xlt('SMS'); ?></a>
+                <?php } ?>
+            </div>
+        </header>
+
+        
+        <?php
+}
+    
+function common_footer($display = "")
+{
+    ?>
+        
+        <div class="fbar M6hT6 As6eLe" style="position:fixed;
+   left:0px;
+   bottom:0px;
+   ">
+            <style>
+                .fbar {
+                    background: #f2f2f2;
+                    line-height: 20px;
+                    padding: 0px 20px;
+                    text-align: center;
+                    margin-left: -27px;
+                    font-size: small;
+                }
+                .fbar a {
+                    text-decoration: none;
+                    white-space: nowrap;
+                    color: #777;
+                }
+
+                .fbar a:hover {
+                    color: #333;
+                }
+                
+                .M6hT6 {
+                    background: #f2f2f2;
+                    left: 0;
+                    right: 0;
+                    -webkit-text-size-adjust: none;
+                }
+                .M6hT6 a {
+                    text-decoration: none;
+                }
+                
+                .As6eLe {
+                    padding-bottom: 5px;
+                }
+                
+                html, body {
+                    height: 100%
+                }
+                .JQyAhb {
+                    border-top: 1px solid #e4e4e4;
+                    padding-top: 10px
+                }
+                .Fx4vi {
+                    padding-left: 27px;
+                    margin: 0 !important
+                }
+
+
+            </style>
+            <div class="JQyAhb" >
+                <span style="display:inline-block;position:relative">
+                    <a class="Fx4vi"
+                       href="https://open-emr.org"><?php echo xlt('openEMR'); ?></a>
+                    <a class="Fx4vi"
+                       href="//community.open-emr.org/"><?php echo xlt('Forum'); ?></a>
+                    <a class="Fx4vi"
+                       href="//chat.open-emr.org/home"><?php echo xlt('Chat'); ?></a>
+                    <a class="Fx4vi"
+                       href="//www.open-emr.org/wiki/index.php/OpenEMR_5.0.1_Users_Guide"><?php echo xlt('User Manuals'); ?></a>
```

Comment:
We have a better way to build manual links. See these lines of code in About script:
https://github.com/openemr/openemr/blob/6cf58036a3eea3aaede3cfffc6e79110d6f1b402/interface/main/about_page.php#L35
https://github.com/openemr/openemr/blob/6cf58036a3eea3aaede3cfffc6e79110d6f1b402/interface/main/about_page.php#L112-L113
https://github.com/openemr/openemr/blob/6cf58036a3eea3aaede3cfffc6e79110d6f1b402/interface/main/about_page.php#L127